### DISCUSSION TITLE
How to change single click for zoom not double click

### DISCUSSION THREAD

**Discussion by GoldenRhynoDev** [igordanchenko/yet-another-react-lightbox#285] [Q&A] [answered]
Hi, nice to meet you.
Now I am using yet-another-react-lightbox and it looks cool.
But I have a question.
How can I change single click for zoom not default double click?
Could you give me your hand?
Thanks.

**Comment by igordanchenko** [comment, marked as answer]
I'm afraid there is no such option. All available zoom related gestures are listed in the documentation - https://yet-another-react-lightbox.com/plugins/zoom

**Reply by GoldenRhynoDev** [reply]
Hi @igordanchenko 
I have already look this documentation.
But I can't find a solution.
So is there no other way to implement this?

**Reply by igordanchenko** [reply]
As I explained above, this library doesn't support what you are asking for.
